doc/whats_new/v0.22.rst

```diff
@@ -445,6 +445,13 @@ Changelog
   with a target matrix `Y` in which the first column was constant.
   :issue:`13609` by :user:`Camila Williamson <camilaagw>`.
 
+:mod:`sklearn.metrics`
+..................................
+
+- |API| Introduced new scorer `neg_brier_loss_score` to better reflect real
```

Comment:
As a scorer, let's make this look like a string. But actually these aren't the right names anyway. The relevant scoring keys are 'neg_brier_score' and 'brier_score_loss'

Reply:
I got confused after this comment https://github.com/scikit-learn/scikit-learn/pull/14123#discussion_r321134299

Reply:
if you mean keys in the `sklearn.metrics.SCORERS` shouldn't they be `neg_brier_score_loss` and `brier_score_loss`?

Reply:
Yes, they should